### ISSUE TITLE
[BUG][Cypress] Missing baseline folder causes failure

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#273]
## What tool(s) are you using?
* cypress-accessibility-checker

## Detailed description
If the baseline folder is missing, it fails. The tools should just return an empty object for the missing folder.

**Comment by ryandudek** [comment]
@tombrunet I'm running into this issue as well, did you find a workaround for it?

**Comment by tombrunet** [comment]
@shunguoy Why did you close this? It was still failing.

**Comment by tombrunet** [comment]
Will be fixed by https://github.com/IBMa/equal-access/pull/818

**Comment by shunguoy** [comment]
@tombrunet  either I closed this by mistake or asked to do so.  I have no clue now why it's closed.
